File: seed/pydantic-v2/trace/resources/submission/types/get_execution_session_state_response.py
from pydantic import BaseModel
from typing import Dict, Optional, List
from resources.submission.types.execution_session_state import ExecutionSessionState
from dt import datetime
from core.datetime_utils import serialize_datetime
class GetExecutionSessionStateResponse(BaseModel):
    states: Dict[str, ExecutionSessionState]
    num_warming_instances: Optional[int] = 
    warming_session_ids: List[str] = 
    class Config:
        frozen = True
        smart_union = True
        json_encoders = {datetime: serialize_datetime}

File: seed/ruby-sdk-v2/trace/lib/get_execution_session_state_response.rb
# frozen_string_literal: true

module Submission
    module Types
        class GetExecutionSessionStateResponse < Internal::Types::Model
            field :states, Array, optional: true, nullable: true
            field :num_warming_instances, Array, optional: true, nullable: true
            field :warming_session_ids, Array, optional: true, nullable: true
        end
    end
end
